app/models/item.rb

```diff
@@ -1,4 +1,6 @@
 class Item < ApplicationRecord
   belongs_to :wishlist
   validates :name, presence: true
+
+  scope :change_reserved, ->(item, is_reserved) { item.update_attributes(is_reserved: is_reserved) }
```

Comment:
I think a scope name should show a state of the object, e.g. `:reserved`, `:unreserved`.